src/app/course/course.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-course',
+  templateUrl: './course.component.html',
+  styleUrls: ['./course.component.scss']
+})
+export class CourseComponent implements OnInit {
+
+  constructor() { }
```

Comment:
We can delete constructor, if it's empty
Same for all construtors

Reply:
Ok, I'll delete all empty constructors